generators/chipyard/src/main/scala/floorplan/ChipTopFloorplans.scala

```diff
@@ -0,0 +1,29 @@
+// See LICENSE for license details
+package chipyard.floorplan
+
+import chipyard.TestHarness
+import chipyard.{ChipTopLazyRawModuleImp, BuildSystem, DigitalTop}
+import freechips.rocketchip.config.{Parameters}
+import barstools.floorplan.chisel.{FloorplanAspect, Floorplan, FloorplanFunction}
+
+object ChipTopFloorplans {
+
+  def default: FloorplanFunction = {
+    case top: ChipTopLazyRawModuleImp =>
```

Comment:
@jwright6323 Is there a way to not have to add the `ChipTopLazyRawModuleImp`? It appears this is the only way get the the `DigitalTop`'s tiles.

Reply:
Can't we do a 
```
top.outer match { 
  case c: ChipTop => c.lazySystem match { 
    case d: DigitalTop => ...
    
    }
  }
}
```

This is admittedly ugly

Reply:
Does `.wrapper` exist? Looking at the implementation of `LazyRawModuleImp`:
```
/** Actual description of a [[Module]] which can be instantiated by a call to [[LazyModule.module]].
  *
  * @param wrapper the [[LazyModule]] from which the `.module` call is being made.
  */
class LazyModuleImp(val wrapper: LazyModule) extends MultiIOModule with LazyModuleImpLike {
  /** Instantiate hardware of this `Module`. */
  val (auto, dangles) = instantiate()
}
```

Reply:
Yes, `.wrapper` exists, and I could do nested matches on `LazyRawModuleImp` followed by `ChipTop` and then `DigitalTop`, but the problem with that is that other `LazyRawModuleImp`s (many other modules in the design) get matched and would be expected to also have a floorplan, which is not what we want. The `ChipTopFloorplans` object should only apply to `ChipTop`.

Reply:
Ok. I see based on how the FloorplanFunction works, what you have here is probably the best solution